### PR TITLE
Update catalan

### PR DESCRIPTION
**Pull request by ivanbea**
New lines added, new lines localized.

Check it please @jordimas @darccio @surfdude29

### CODE REVIEW

File: src/locale/locales/ca/messages.po [hunk 1]
```diff
@@ -458,7 +458,7 @@ msgstr "Afegeix els canals recomanats"
 
 #: src/screens/StarterPack/Wizard/index.tsx:488
 msgid "Add some feeds to your starter pack!"
-msgstr ""
+msgstr "Afegiu persones que creus que els agradaria seguir als altres al teu starter pack"
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:461]: This looks like it's for a different string?

File: src/locale/locales/ca/messages.po [hunk 2]
```diff
@@ -2224,7 +2224,7 @@ msgstr "Ensenyament"
 
 #: src/components/dialogs/ThreadgateEditor.tsx:98
 msgid "Either choose \"Everybody\" or \"Nobody\""
-msgstr ""
+msgstr "Tria \"Tothom\" o \"Ningú\"
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:2227]:
```suggestion
msgstr "Tria \"Tothom\" o \"Ningú\""
```
There's a quotation mark missing at the end here

File: src/locale/locales/ca/messages.po [hunk 3]
```diff
@@ -3338,23 +3338,23 @@ msgstr "Codis d'invitació: 1 disponible"
 
 #: src/components/StarterPack/ShareDialog.tsx:96
 msgid "Invite people to this starter pack!"
-msgstr ""
+msgstr "Convida a gent a aquest starter pack""
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:3341]:
```suggestion
msgstr "Convida a gent a aquest starter pack!"
```
There's a missing `!` and an extra `"` at the end here

File: src/locale/locales/ca/messages.po [hunk 4]
```diff
@@ -3338,23 +3338,23 @@ msgstr "Codis d'invitació: 1 disponible"
...
 
 #: src/screens/StarterPack/Wizard/StepDetails.tsx:35
 msgid "Invite your friends to follow your favorite feeds and people"
-msgstr ""
+msgstr "Convida els teus amics a seguir els teus canals i persones preferides"
 
 #: src/screens/StarterPack/Wizard/StepDetails.tsx:32
 msgid "Invites, but personal"
-msgstr ""
+msgstr "Convida a bluesky de manera més personalitzada"
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:3349]:
```suggestion
msgstr "Convida a Bluesky de manera més personalitzada"
```
I think `Bluesky` should be capitalised here

File: src/locale/locales/ca/messages.po [hunk 5]
```diff
@@ -3363,11 +3363,11 @@ msgstr "Feines"
 #: src/screens/StarterPack/StarterPackLandingScreen.tsx:200
 #: src/screens/StarterPack/StarterPackLandingScreen.tsx:206
 msgid "Join Bluesky"
-msgstr ""
+msgstr "Uneix-te a Bluesky"
 
 #: src/components/StarterPack/QrCode.tsx:56
 msgid "Join the conversation"
-msgstr ""
+msgstr "Uneix´-te a la conversa"
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:3370]:
```suggestion
msgstr "Uneix-te a la conversa"
```
There's a stray `´` here

File: src/locale/locales/ca/messages.po [hunk 6]
```diff
@@ -6732,7 +6732,7 @@ msgstr "Aquest identificador ja està agafat."
 #: src/screens/StarterPack/Wizard/index.tsx:106
 #: src/screens/StarterPack/Wizard/index.tsx:114
 msgid "That starter pack could not be found."
-msgstr ""
+msgstr "No s'ha pogut trobar aquest starter pack"
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:6735]:
```suggestion
msgstr "No s'ha pogut trobar aquest starter pack."
```
There's a missing `.` at the end

File: src/locale/locales/ca/messages.po (hunk 1 above)
Comment (ivanbea) [reply src/locale/locales/ca/messages.po]: Thank you very much for all your corrections!

**Review by pfrazee — APPROVED** [review]
Awesome, thanks everyone!

**Pull request merged**